graphql-server/src/schemas/neo4j.ts

```diff
@@ -247,7 +494,209 @@ function buildResolvers(
         };
       },
     },
+
+    // async samples(
+    //   _source: undefined,
+    //   args: any,
+    //   { samplesLoader }: ApolloServerContext
+    // ) {
+
     Query: {
+      // THIS ONE CAN BE CALLED FROM THE BROWSER
+      async sampleDashboardQuery(
```

Comment:
I was calling this from the browser with the following, but in the code we use the above method. Don't leave both!
query ExampleQuery { sampleDashboardQuery { primaryId }}